eppo_core/src/ufc/models.rs

```diff
@@ -88,24 +97,25 @@ pub enum Value {
 }
 
 impl Value {
-    pub fn to_assignment_value(&self, ty: VariationType) -> Option<AssignmentValue> {
+    /// Try to convert `Value` to [`AssignmentValue`] under the given [`VariationType`].
+    pub(crate) fn to_assignment_value(&self, ty: VariationType) -> Option<AssignmentValue> {
         Some(match ty {
             VariationType::String => AssignmentValue::String(self.as_string()?.to_owned()),
             VariationType::Integer => AssignmentValue::Integer(self.as_integer()?),
             VariationType::Numeric => AssignmentValue::Numeric(self.as_number()?),
             VariationType::Boolean => AssignmentValue::Boolean(self.as_boolean()?),
-            VariationType::Json => AssignmentValue::Json(self.as_json()?),
+            VariationType::Json => AssignmentValue::Json(self.to_json()?),
```

Comment:
noticed json had `to` instead of `as`--is this some rust nuance?

Reply:
It's a naming convention. `as` means it's a simple cast (essentially free). `to` suggests that there's some computation involved (parsing JSON in this case).

https://rust-lang.github.io/api-guidelines/naming.html#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv